build-system/exec.js

```diff
@@ -46,6 +59,16 @@ exports.exec = function(cmd) {
 }
 
 /**
+ * Starts off the provided command in a background process.
+ *
+ * @param {string} cmd Command line to execute.
+ */
+exports.execAsync = function(cmd) {
+  let p = spawnProcessAsync(cmd, {'stdio': 'inherit', 'detached': 'true'});
```

Comment:
provide a way for the invoker to know when this is completed whether its returning a promise or a callback api or returning the stream so the invoker can at least listen to an event